From 639aaaa35c2246760f2e2ee2d55f27c24bb95cd7 Mon Sep 17 00:00:00 2001
From: hwg <hwg@rock-chips.com>
Date: Wed, 4 Mar 2015 10:49:13 +0800
Subject: [PATCH] support different wifi bt chip auto compatible, modify 
 arch/arm64/boot/dts/rk3368-p9_818.dts, arch/arm64/boot/dts/rk3368-tb_8846.dts

---
 arch/arm64/boot/dts/rk3368-p9_818.dts  | 15 +++++++--------
 arch/arm64/boot/dts/rk3368-tb_8846.dts | 13 ++++++-------
 2 files changed, 13 insertions(+), 15 deletions(-)

diff --git a/arch/arm64/boot/dts/rk3368-p9_818.dts b/arch/arm64/boot/dts/rk3368-p9_818.dts
index a2addcf15800..15541879e1df 100755
--- a/arch/arm64/boot/dts/rk3368-p9_818.dts
+++ b/arch/arm64/boot/dts/rk3368-p9_818.dts
@@ -13,14 +13,13 @@
 		compatible = "wlan-platdata";
 
 		rockchip,grf = <&grf>;
-		
-		/* wifi_chip_type - wifi chip define
-		 * bcmwifi ==> like ap6xxx, rk90x;
-		 * rtkwifi ==> like rtl8188xx, rtl8723xx;
-		 * esp8089 ==> esp8089;
-		 * other   ==> for other wifi;
-		 */
-		wifi_chip_type = "bcmwifi";
+
+                /* wifi_chip_type - wifi chip define
+                 * ap6210, ap6330, ap6335
+                 * rtl8188eu, rtl8723bs, rtl8723bu
+                 * esp8089
+                */
+                wifi_chip_type = "ap6210";		
 
 		sdio_vref = <1800>; //1800mv or 3300mv
  		power_pmu_regulator = "rk818_ldo8_reg";
diff --git a/arch/arm64/boot/dts/rk3368-tb_8846.dts b/arch/arm64/boot/dts/rk3368-tb_8846.dts
index 665092fc8618..d0cf47dae3ca 100644
--- a/arch/arm64/boot/dts/rk3368-tb_8846.dts
+++ b/arch/arm64/boot/dts/rk3368-tb_8846.dts
@@ -16,13 +16,12 @@
 		compatible = "wlan-platdata";
                 rockchip,grf = <&grf>;
 
-		/* wifi_chip_type - wifi chip define
-		 * bcmwifi ==> like ap6xxx, rk90x;
-		 * rtkwifi ==> like rtl8188xx, rtl8723xx;
-		 * esp8089 ==> esp8089;
-		 * other   ==> for other wifi;
-		 */
-		wifi_chip_type = "bcmwifi";
+                /* wifi_chip_type - wifi chip define
+                 * ap6210, ap6330, ap6335
+                 * rtl8188eu, rtl8723bs, rtl8723bu
+                 * esp8089
+                */
+                wifi_chip_type = "ap6335";
 
 		sdio_vref = <1800>; //1800mv or 3300mv
 
-- 
2.35.3

